delegator/build-all.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-cargo +stable contract build --manifest-path accumulator/Cargo.toml
-cargo +stable contract build --manifest-path adder/Cargo.toml
-cargo +stable contract build --manifest-path subber/Cargo.toml
-cargo +stable contract build
+cargo +stable contract build --manifest-path accumulator/Cargo.toml --release
```

Comment:
`+stable` can be removed as it is enabled by default

Reply:
done